pkg/task/restore.go

```diff
@@ -467,3 +462,18 @@ func RunRestoreTiflashReplica(c context.Context, g glue.Glue, cmdName string, cf
 	summary.SetSuccessStatus(true)
 	return nil
 }
+
+func enableTiDBConfig() {
+	// set max-index-length before execute DDLs and create tables
+	// we set this value to max(3072*4), otherwise we might not restore table
+	// when upstream and downstream both set this value greater than default(3072)
+	conf := config.GetGlobalConfig()
+	conf.MaxIndexLength = config.DefMaxOfMaxIndexLength
+	log.Warn("set max-index-length to max(3072*4) to skip check index length in DDL")
```

Comment:
Perhaps this warning should also mention the experimental features. But I'm not sure about the point of having such warning at all when it's always emitted @5kbpers.